tests/test_asg.py

```diff
@@ -463,6 +465,51 @@ def test_asg_security_group(self):
         self.assertEqual(len(resources), 1)
         self.assertEqual(resources[0]['AutoScalingGroupName'], 'ContainersFTW')
 
+    def test_asg_activity_filter(self):
+        self.patch(LaunchActivityFilter, 'executor_factory', MainThreadExecutor)
+        session = self.replay_flight_data('test_asg_launch_activity_filter')
+        p = self.load_policy({
+            'name': 'asg-activities',
+            'resource': 'asg',
+            'filters': [{
+                "or": [
+                    {'type': 'launch-activity', 'status': ['Successful']},
+                    {'type': 'launch-activity', 'status': ['Failed']}
+                ]}]}, session_factory=session)
+        resources = p.run()
+        self.assertEqual(len(resources), 2)
+        self.assertEqual(
+            sorted([r['AutoScalingGroupName'] for r in resources]),
+            ['c7n-asg-launch-failure', 'c7n-asg-launch-success'])
+
+    def test_asg_activity_failed_filter(self):
+        self.patch(LaunchActivityFilter, 'executor_factory', MainThreadExecutor)
+        session = self.replay_flight_data('test_asg_launch_activity_failure')
+        p = self.load_policy({
+            'name': 'failed-asg-build',
+            'resource': 'asg',
+            'filters': [
```

Comment:
can we drop two of these tests, there not really adding any value from a test coverage perspective, and they bloat the recorded json for this pr.